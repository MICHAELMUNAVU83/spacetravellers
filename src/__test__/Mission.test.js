import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Mission from '../componets/Mission';
import store from '../redux/configureStore';

it('Missiojns renders correctly', () => {
  const tree = TestRenderer.create(
    <Provider store={store}>
      <Mission />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
