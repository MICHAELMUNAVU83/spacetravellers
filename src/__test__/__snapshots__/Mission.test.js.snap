// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missiojns renders correctly 1`] = `
<div>
  <table
    className="table table-bordered"
  >
    <thead>
      <tr>
        <th
          scope="col"
        >
          Mission
        </th>
        <th
          scope="col"
        >
          Description
        </th>
        <th
          scope="col"
        >
          Status
        </th>
        <th
          scope="col"
        >
           
        </th>
      </tr>
    </thead>
  </table>
</div>
`;
